tests/sentry/workflow_engine/models/test_workflow.py

```diff
@@ -0,0 +1,19 @@
+from sentry.testutils.cases import TestCase
+from sentry.workflow_engine.models.workflow import Workflow
+
+
+class WorkflowTest(TestCase):
+    def setUp(self):
+        self.org = self.create_organization()
+
+    def tearDown(self):
+        self.org.delete()
+
+    def test_create_simple_workflow(self):
+        workflow = Workflow.objects.create(name="test_workflow", organization=self.org)
+
+        workflow.save()
+        workflow.refresh_from_db()
+
+        assert workflow.name == "test_workflow"
+        assert workflow.organization == self.org
```

Comment:
imo you don't really need to test the orm, but up to you

Reply:
yeah, agree, i just wanted a placeholder test with an import / example creation so when i go to start work on the next bit, it's halfway done for me. (i also wanted to confirm that i had setup all the stuff correctly 🎉)